x-pack/plugins/infra/public/alerting/log_threshold/components/expression_editor/editor.tsx

```diff
@@ -90,7 +90,7 @@ const createDefaultRatioAlertParams = (
 });
 
 export const ExpressionEditor: React.FC<
-  AlertTypeParamsExpressionProps<PartialAlertParams, LogsContextMeta>
+  AlertTypeParamsExpressionProps<PartialRuleParams, LogsContextMeta>
```

Comment:
AlertTypeParamsExpressionProps will be refactored with a dedicated PR for triggers_actions_ui plugin.